### PR TITLE
Create plot: Coulomb law

### PR DESCRIPTION
**Pull request by Voldoned**

### CODE REVIEW

File: plots/laws/electricity/coulomb_force_for_electron.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    coulomb_law.second_charge: CHARGE_OF_ELECTRON,
+    coulomb_law.distance: distance,
+    coulomb_law.units.coulomb_constant:
+    convert_to(Quantity(units.coulomb_constant),
+               units.newton * (units.meters / units.coulomb) ** 2)
+}).rhs
+
+p1 = plot(force_equation, (distance, 1, 5),
+          line_color="blue",
+          title="Coluomb Law",
```
Comment (blackyblack) [inline plots/laws/electricity/coulomb_force_for_electron.py:27]:
```suggestion
          title="Coulomb Law",
```

File: plots/laws/electricity/coulomb_force_for_electron.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+distance = Symbol("dictance")
+electrostatic_constant = Symbol("electrostatic_constant")
+
+force_equation = coulomb_law.law.subs({
+    coulomb_law.first_charge: CHARGE_OF_ELECTRON,
+    coulomb_law.second_charge: CHARGE_OF_ELECTRON,
+    coulomb_law.distance: distance,
+    coulomb_law.units.coulomb_constant:
+    convert_to(Quantity(units.coulomb_constant),
```
Comment (blackyblack) [inline plots/laws/electricity/coulomb_force_for_electron.py:21]: You can simply put a number here. For plot it's OK.

File: plots/laws/electricity/coulomb_force_for_electron.py [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+               units.newton * (units.meters / units.coulomb) ** 2)
+}).rhs
+
+p1 = plot(force_equation, (distance, 1, 5),
+          line_color="blue",
+          title="Coluomb Law",
+          xlabel="r, m",
+          ylabel="F, N",
+          backend=MatplotlibBackend,
+          legend=True,
```
Comment (blackyblack) [inline plots/laws/electricity/coulomb_force_for_electron.py:31]: Something is wrong with the plot legend. Make sure it shows proper description of the graphic.

Comment (Voldoned) [reply plots/laws/electricity/coulomb_force_for_electron.py:30]: I forget to add parameter 'label'. Now is added.

File: plots/laws/electricity/coulomb_force_for_electron.py [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+from symplyphysics import (print_expression, units, Quantity, convert_to)
+
+from symplyphysics.laws.electricity import force_from_charge_and_distance as coulomb_law
+
+ELECTROSTATIC_CONSTANT = 9 * 1E9
+CHARGE_OF_ELECTRON = -1.6 * 1E-19
+
+print(f"Formula is:\n{coulomb_law.print_law()}")
+
+distance = Symbol("dictance")
```
Comment (blackyblack) [inline plots/laws/electricity/coulomb_force_for_electron.py:14]:
```suggestion
distance = Symbol("distance")
```

File: plots/laws/electricity/coulomb_force_for_electron.py [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+    coulomb_law.units.coulomb_constant: ELECTROSTATIC_CONSTANT
+}).rhs
+
+p1 = plot(force_equation, (distance, 1, 5),
+          line_color="blue",
+          title="Coulomb Law",
+          xlabel="r, m",
+          ylabel="F, N",
+          backend=MatplotlibBackend,
+          label="The force acting on an electron from another electron",
```
Comment (blackyblack) [inline plots/laws/electricity/coulomb_force_for_electron.py:29]: It's too long. Use `Electrostatic force`

**Review by blackyblack — APPROVED** [review]

**Pull request merged**